radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/substatehash/BerkeleySubstateAccumulatorHashStore.java

```diff
@@ -137,23 +138,24 @@ public class BerkeleySubstateAccumulatorHashStore implements BerkeleyAdditionalS
 
   private final Stopwatch timeSpentOnSubstateAccumulatorThisEpoch = Stopwatch.createUnstarted();
 
-  private Forks forks;
   private final Provider<RadixEngine<LedgerAndBFTProof>> radixEngineProvider;
 
   private Writer epochsHashFileWriter;
   private BufferedReader epochsHashFileBufferedReader;
 
-  private boolean isUpdateEpochHashFileEnabled;
-  private boolean isVerifyEpochHashEnabled;
+  private final Network currentNetwork;
```

Comment:
I would just call it `network`, it cannot change while the app is running so `current` is meaningless

Reply:
Done